packages/epics/src/agreements/components/agreement-card.tsx

```diff
@@ -18,6 +18,7 @@ type AgreementCardProps = {
   views?: number;
   comments?: CardCommentProps[];
   isLoading?: boolean;
+  withoutAvatar?: boolean;
```

Comment:
Why not just check `creator` prop itself? And otherwise I'd prefer to use `hasAvatar` instead of the inverted version.

Reply:
Fixed